docs/titan-reference/button-reference.md

```diff
@@ -0,0 +1,485 @@
+---
+id: button-reference
+title: Button reference
+sidebar_label: Button reference
+---
+
+This section lists all the physical buttons and tells you what they do. The **links** take you to the section of the manual where the function is explained in more detail.
+
+``<n>`` means to type in a numeric value on the keypad.
+
+\<View\> \<Colour\> indicates to press a sequence of buttons.
+
+\<Avo\> + \<Clear\> with a plus symbol means to hold down the first button while pressing the second button.
+
+{Select} means to press a playback select button.
+
+---
+
+ ## \<@\> 
+ (by numeric keypad) Allows numeric entry of attribute values. For the \<@\> buttons next to the wheels see [Wheel @](#wheel-).
+
+ Example: \<Fixture\> 1 \<And\> 5 \<And\> 7 \<@\> 75 \<Enter\>
+
+  Buttons | Action
+ --------|--------
+ \<@\> ``<n>`` \<Enter\> | Set intensity of currently selected fixtures to ``<n>`` in the programmer. (``<n>`` normally 0-99, optionally can be single digit 0-9, see [Formatting](../system-settings/user-settings.md#formatting))
+ \<@\> \<@\> | Set intensity of currently selected fixtures to full in the programmer.
+  
+ --- 
+## \<Align\> 
+ (D9 only) Allows copying of attributes between fixtures. On consoles other than D9 this function is accessed using \<ML Menu\>. See [Align Fixtures](../controlling-fixtures/changing-fixture-attributes.md/#align-fixtures).
+
+ --- 
+## \<All\> 
+ Used to start and end odd/even selection from a range of selected fixtures. See [Selecting fixtures using a pattern](../controlling-fixtures.md/#selecting-fixtures-using-a-pattern).
+
+ --- 
+## \<And\> 
+ Used when selecting multiple items using the numeric keypad, can be combined with \<Through\> and \<Not\>. See [Selecting fixtures and dimmers by number](../controlling-fixtures.md/#selecting-fixtures-and-dimmers-by-number-channel). Some consoles do not have this button, an \[And\] softkey is also provided.
+
+ Example:  \<Fixture\> 1 \<And\> 5 \<And\> 7
+
+ --- 
+## \<Assign\> 
+ (D9 and ST only) Allows you to reassign attributes controlled by the trackball and (on D9 only) intensity wheel. See [Trackball](../controlling-fixtures/changing-fixture-attributes.md#trackball-diamond-9-and-sapphire-touch-only).
+
+ --- 
+## \<Avo\> 
+ Acts as a "Shift" button when held down, giving access to different functions. 
+
+  Buttons | Action
+ --------|--------
+ \<Avo\> + \<All\> | Set zero intensity into programmer for unselected fixtures (same as \<Rem Dim\>), see [Turn off unselected fixtures](../controlling-fixtures.md#turn-off-unselected-fixtures-remainder-dim). Also used when setting Attribute Times to deselect all fixtures, see [Attribute Fade Times](../cue-lists/cue-list-timing.md#individual-attribute-fade-times).
```

Comment:
Some more:
`<Avo><Undo>` - Redo
`<Avo><Back>` - Undo
`<Avo><@>` - Redo
`<Avo><Highlight>` - Record highlight
`<Avo><Flash On>` - Flash selected fixtures out (actually I think `<Flash On>` is missing entirely?
`<Avo><Release>` - Release all playbacks by priority
`<Avo><Fix +/-`> - Jump pattern forward backward in pattern select mode
`<Avo><Min/Max>` - Select different active window
`<Avo>2/4/6/8` - Cursor up, left, right, down
`<Avo><Shape>` - Same as `<FX>` on Diamond (enter shape+FX menu)

Reply:
How does "Release all playbacks by priority" work - is that the same as \<release\> \<release\> .

Reply:
Yes, it's the same action as `<Release><Release>`